src/assets/icons/ic-success-large.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="56" height="56" viewBox="0 0 56 56" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
File name could be like `ic-success-with-light-background.svg` or something  more specific, as we already have `ic-success.svg`